### PR TITLE
Audit

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Editing audit logs page to reflect that Azure is not supported. This is a request from the field as it is confusing clients

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/collaborate/manage-access/audit-log.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ description: "You can troubleshoot possible issues and provide security audits b
 sidebar_label: "Audit log"
 ---
 
+:::note
+
+Audit logs are currently not available for Microsoft Azure customers.
```
Comment (matthewshaver) [inline website/docs/docs/collaborate/manage-access/audit-log.md]:
```suggestion
Audit logs are currently not supported for Microsoft Azure.
```

File: website/docs/docs/collaborate/manage-access/audit-log.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ description: "You can troubleshoot possible issues and provide security audits b
 sidebar_label: "Audit log"
 ---
 
+:::note
+
+Audit logs are currently not supported for Microsoft Azure.
+
+:::
+
 To review actions performed by people in your organization, dbt provides logs of audited user and system events in real time. The audit log appears as events happen and includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events.
```
Comment (nghi-ly) [inline website/docs/docs/collaborate/manage-access/audit-log.md]: suggest moving the callout from line 8 to here on line 15

**Review by nghi-ly — COMMENTED** [review]
looks good! left one comment re: placement

File: website/docs/docs/collaborate/manage-access/audit-log.md (hunk 2 above)
Comment (matthewshaver) [reply website/docs/docs/collaborate/manage-access/audit-log.md]: Good call! Taken care of it

**Review by nghi-ly — APPROVED** [review]
🚢

**Pull request merged**